pyeudiw/satosa/default/openid4vp_backend.py

```diff
@@ -200,7 +200,11 @@ def get_response_endpoint(self, context: Context) -> Response:
         self._log_function_debug("get_response_endpoint", context)
 
         state = context.qs_params.get("id", None)
-        session_id = context.state["SESSION_ID"]
+        session_id = context.state.get("SESSION_ID", None)
+
+        if not state:
+            return self._handle_400(context, "No state found in query params")
```

Comment:
SESSION_ID is not provided in the query params but in the internal storage instance bound to the cookie brought by the user-agent

please say that the SESSION ID is not found or something like this